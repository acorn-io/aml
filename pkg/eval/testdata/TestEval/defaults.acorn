foo: default 1
define bar: {
	foo: default 1
	baz: string || 3
}

baz: foo + 2
blah: bar({
	foo: 2
	baz: "hello"
})
blah2: bar({
	foo: 2
})
blah3: bar({
	baz: 3
})
