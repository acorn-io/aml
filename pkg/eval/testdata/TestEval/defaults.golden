{
  "baz": 3,
  "blah": {
    "baz": "hello",
    "foo": 2
  },
  "blah2": {
    "baz": 3,
    "foo": 2
  },
  "blah3": {
    "baz": 3,
    "foo": 1
  },
  "foo": 1
}